pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
Yeah, that's actually a good example-- if there's groups that are half-baked and we have client (or server) code for them but we definitely don't want it running in production. I can also imagine wanting to inject a custom codec, although that's a bit more contrived.